src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -16,13 +16,13 @@ folder InstallDir:\MSBuild\Current
 
 folder InstallDir:\MSBuild\Current\Bin
   file source=$(MSBuildConversionBinPath)Microsoft.Build.Conversion.Core.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)Microsoft.Build.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)Microsoft.Build.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
   file source=$(MSBuildConversionBinPath)Microsoft.Build.Engine.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)Microsoft.Build.Framework.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)Microsoft.Build.Framework.tlb
-  file source=$(X86BinPath)Microsoft.Build.Tasks.Core.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)Microsoft.Build.Utilities.Core.dll vs.file.ngenArchitecture=all
-  file source=$(X86BinPath)MSBuild.exe vs.file.ngenArchitecture=x86
+  file source=$(X86BinPath)Microsoft.Build.Tasks.Core.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
+  file source=$(X86BinPath)Microsoft.Build.Utilities.Core.dll vs.file.ngenArchitecture=all vs.file.ngenPriority=1
+  file source=$(X86BinPath)MSBuild.exe vs.file.ngenArchitecture=x86 vs.file.ngenPriority=1
```

Comment:
What VS scenarios are you aiming for with this? I would expect this to be used primarily later in solution load and for build, so it might not need to be higher-pri.

Reply:
We're targeting startup, open solution, and build.

Reply:
I have a proposal on it. Is it OK to link to internal docs in PRs?

Reply:
Wouldn't be the first time, but you can also email/IM me if you'd rather.